accountsplus/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '1.3.1'
+__version__ = '1.6.0'
```

Comment:
@waseem-omar I was testing something and I thought there was an issue with my version. It should not have been included. Will remove it